### DISCUSSION TITLE
[Node Operator Question] <Title>My node has been behind 20000+ blocks for a week

### DISCUSSION THREAD

**Discussion by wetezos** [ethereum-optimism/developers#321] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

My node has been behind 20000+ blocks for a week, my server :10 cores, 128g memory, 4T SSD disk
<img width="1727" alt="op" src="https://github.com/ethereum-optimism/developers/assets/40154568/3eb6a491-5d69-425c-887e-697f2f187b6b">

### Protocol Description

op-geth:v1.101308.1
op-node:v1.6.1
network:mainnet

### Node Logs

docker logs simple-optimism-node-op-geth-1 -f  --tail 20
`INFO [03-27|15:25:17.833] Starting work on payload                 id=0xec461b6bee1f36f2
INFO [03-27|15:25:17.836] Imported new potential chain segment     number=117,955,011 hash=75a6dd..08d998 blocks=1 txs=1 mgas=0.044 elapsed="995.621µs" mgasps=44.044  age=12h18m38s snapdiffs=1.84MiB    triedirty=7.70MiB
INFO [03-27|15:25:17.836] Chain head was updated                   number=117,955,011 hash=75a6dd..08d998 root=c7b6a0..52b9ad elapsed="60.121µs"  age=12h18m38s
INFO [03-27|15:25:17.839] Starting work on payload                 id=0x3f7241f74cf68636
INFO [03-27|15:25:17.842] Imported new potential chain segment     number=117,955,012 hash=771f83..dabad8 blocks=1 txs=1 mgas=0.044 elapsed=1.007ms     mgasps=43.521  age=12h18m36s snapdiffs=1.84MiB    triedirty=7.70MiB
INFO [03-27|15:25:17.842] Chain head was updated                   number=117,955,012 hash=771f83..dabad8 root=917403..f1d3d7 elapsed="69.729µs"  age=12h18m36s
INFO [03-27|15:25:17.845] Starting work on payload                 id=0xde5aa9319aa68d51
INFO [03-27|15:25:17.847] Imported new potential chain segment     number=117,955,013 hash=f8031f..909a27 blocks=1 txs=1 mgas=0.044 elapsed="981.936µs" mgasps=44.658  age=12h18m34s snapdiffs=1.84MiB    triedirty=7.70MiB
INFO [03-27|15:25:17.848] Chain head was updated                   number=117,955,013 hash=f8031f..909a27 root=313847..a95b38 elapsed="77.894µs"  age=12h18m34s
INFO [03-27|15:25:17.851] Starting work on payload                 id=0x465d447483de0bdb
INFO [03-27|15:25:17.853] Imported new potential chain segment     number=117,955,014 hash=e77eb3..7f5583 blocks=1 txs=1 mgas=0.044 elapsed="949.736µs" mgasps=46.172  age=12h18m32s snapdiffs=1.84MiB    triedirty=7.70MiB
INFO [03-27|15:25:17.854] Chain head was updated                   number=117,955,014 hash=e77eb3..7f5583 root=829780..266731 elapsed="72.775µs"  age=12h18m32s`

**Comment by imtipi** [comment]
can you post your .env file?

**Reply by wetezos** [reply]
Sure

###############################################################################
#                                ↓ REQUIRED ↓                                 #
###############################################################################

# Network to run the node on ("op-mainnet" or "op-sepolia")
NETWORK_NAME=op-mainnet

# Type of node to run ("full" or "archive"), note that "archive" is 10x bigger
NODE_TYPE=full

###############################################################################
#                            ↓ REQUIRED (BEDROCK) ↓                           #
###############################################################################

# Where to get the Bedrock database ("download" or "migration"), see README
BEDROCK_SOURCE=download

# L1 node that the op-node (Bedrock) will get chain data from
OP_NODE__RPC_ENDPOINT=https://mainnet-rpc.wetez.io/eth/v1/5646ccxxxxx

# L1 beacon endpoint, you can setup your own or use Quicknode
OP_NODE__L1_BEACON=https://beaconcha-rpc.wetez.io/eth/v1/5646ccxxxxx

# Type of RPC that op-node is connected to, see README
OP_NODE__RPC_TYPE=basic

# Reference L2 node to run healthcheck against
HEALTHCHECK__REFERENCE_RPC_PROVIDER=https://opt-mainnet.g.alchemy.com/v2/UMv_b5ofrDKiPwAxxxxxx

###############################################################################
#                            ↓ OPTIONAL (BEDROCK) ↓                           #
###############################################################################

# Optional provider to serve legacy RPC requests, see README
OP_GETH__HISTORICAL_RPC=https://mainnet.optimism.io

###############################################################################
#                                ↓ OPTIONAL ↓                                 #
###############################################################################

# Feel free to customize your image tag if you want, uses "latest" by default
# See here for all available images: https://hub.docker.com/u/ethereumoptimism
IMAGE_TAG__L2GETH=
IMAGE_TAG__DTL=
IMAGE_TAG__HEALTCHECK=
IMAGE_TAG__PROMETHEUS=
IMAGE_TAG__GRAFANA=
IMAGE_TAG__INFLUXDB=
IMAGE_TAG__OP_GETH=
IMAGE_TAG__OP_NODE=

# Exposed server ports (must be unique)
# See docker-compose.yml for default values
PORT__L2GETH_HTTP=
PORT__L2GETH_WS=
PORT__DTL=
PORT__HEALTHCHECK_METRICS=
PORT__PROMETHEUS=
PORT__GRAFANA=
PORT__INFLUXDB=
PORT__TORRENT_UI=
PORT__TORRENT=
PORT__OP_GETH_HTTP=
PORT__OP_GETH_WS=
PORT__OP_NODE_P2P=

**Reply by Chomtana** [reply]
I think you need to upgrade the op-node to v1.7.2.

Would you mind pulling and upgrading the node to see if the issue persist?

https://github.com/smartcontracts/simple-optimism-node

```
git pull
docker compose pull
docker compose up -d --build --force-recreate
```

**Comment by wetezos** [comment]
> I think you need to upgrade the op-node to v1.7.2.
> 
> Would you mind pulling and upgrading the node to see if the issue persist?
> 
> https://github.com/smartcontracts/simple-optimism-node
> 
> ```
> git pull
> docker compose pull
> docker compose up -d --build --force-recreate
> ```

Thanks a lot, The node is now running normally
